subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -5042,9 +5043,6 @@ static inline void ctrl_tx_ack(struct ll_conn *conn, struct node_tx **tx,
 			phys = conn->llcp_phy.tx | lll->phy_tx;
 			lll->phy_tx_time = phy_tx_time[phys];
 		}
-
-		/* resume data packet tx */
-		conn->llcp_phy.pause_tx = 0U;
```

Comment:
Where is the pause_tx clear corresponding to this, after its removal?

Reply:
It is below in line 5053.

Now I get it, the reset is present only for master as it sends PDU_DATA_LLCTRL_TYPE_PHY_UPD_IND. Slave should reset it on reception, let me send an update to this PR.

Reply:
> It is below in line 5053.
> 
> Now I get it, the reset is present only for master as it sends PDU_DATA_LLCTRL_TYPE_PHY_UPD_IND. I have remove the regression commit in this PR.

I can confirm that the regression seen with EBQ is now gone.